openlibrary/core/lists/model.py

```diff
@@ -198,7 +198,7 @@ def get_export_list(self) -> dict[str, list[dict]]:
         things = cast(
             list[Thing],
             web.ctx.site.get_many(
-                [seed.key for seed in self.seeds if isinstance(seed, Thing)]
+                [seed for seed in self._get_seed_strings() if seed.startswith("/")]
```

Comment:
Great catch,

@cdrini suggests
```suggestion
                [seed.key for seed in self.get_seeds() if seed._type != "subject"]
```